### PR TITLE
feat: add main file

### PR DESCRIPTION
**Pull request by deri561**
add isOdd function number

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+// wrong function is odd checker
+function isOdd(number) {
+    return number % 2 == 0;
```
Comment (Deri-Kurniawan) [inline main.js]: This is checking `even` number, you should use `!=` or `!==`

File: main.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// wrong function is odd checker
```
Comment (Deri-Kurniawan) [inline main.js]: remove this single comment after fixing the function below correctly

**Review by Deri-Kurniawan — CHANGES_REQUESTED** [review]
Thank you for your work, but I have 2 notes for your code

File: main.js (hunk 2 above)
Comment (deri561) [reply main.js]: Resolved

**Review by deri561 — COMMENTED** [review]
All notes resolved

**Review by Deri-Kurniawan — APPROVED** [review]
Well Done, Good Job!

**Pull request merged**